diff -Naur ./libinput-config-default/meson.build ./libinput-config-default-patched/meson.build
--- ./libinput-config-default/meson.build	2022-06-18 11:55:12.000000000 -0500
+++ ./libinput-config-default-patched/meson.build	2022-07-29 11:05:53.672884659 -0500
@@ -42,14 +42,6 @@
 		install: true,
 		install_dir: '/etc/fish/conf.d'
 	)
-	
-	run_target('pre-uninstall', command: ['/bin/true'])
-else
-	meson.add_install_script('scripts/preload.sh', libdir, project_name)
-	
-	run_target('pre-uninstall',
-		command: ['scripts/unpreload.sh', libdir, project_name]
-	)
 endif
 
 shared_library('input-config',
